src/components/data/FileBrowser.js

```diff
@@ -248,6 +249,13 @@ const BucketBrowserTable = ({
                             },
                             [label]
                           ),
+                          h(ClipboardButton, {
+                            'aria-label': `Copy ${object.name} to clipboard`,
```

Comment:
Let's add a tooltip for this button. The tooltip will also serve as the label.

```suggestion
                            tooltip: 'Copy file URL to clipboard',
```